sensing/preprocessor/gnss/gnss_poser/src/gnss_poser_core.cpp

```diff
@@ -265,13 +250,14 @@ bool GNSSPoser::getTransform(
 
   try {
     *transform_stamped_ptr =
-      tf2_buffer_.lookupTransform(target_frame, source_frame, ros::Time(0), ros::Duration(0.0));
+      tf2_buffer_.lookupTransform(target_frame, source_frame, tf2::TimePointZero);
   } catch (tf2::TransformException & ex) {
-    ROS_WARN_STREAM_THROTTLE(1, ex.what());
-    ROS_WARN_STREAM_THROTTLE(
-      1, "Please publish TF " << target_frame.c_str() << " to " << source_frame.c_str());
+    RCLCPP_WARN_STREAM_THROTTLE(this->get_logger(), clock_, 1, ex.what());
+    RCLCPP_WARN_STREAM_THROTTLE(
+      this->get_logger(), clock_, 1,
+      "Please publish TF " << target_frame.c_str() << " to " << source_frame.c_str());
 
-    transform_stamped_ptr->header.stamp = ros::Time::now();
+    transform_stamped_ptr->header.stamp = rclcpp::Node::now();
```

Comment:
One of those methods that are [not documented](https://docs.ros2.org/dashing/api/rclcpp/classrclcpp_1_1Node.html#af706b231620f1c120b7ccd738ec31867) 
I'm not sure which time it returns, I"m guessing it's the system time as it seems to be a static method. If you respect @nnmm 's comment above, then I think you should ask the `clock_` member for `now()` to serve the sim and replay use cases as well

Reply:
Here is a thread which I had seen in the past go [over the distinction between node and clock](https://github.com/ros2/rclcpp/issues/595). So just to confirm I was under the impression that `rclcpp::Node::now()` will return the current ROS time rather than the current system time?

Reply:
That is correct, they return the same time. I'm fine with using it, it's just strange that you're calling it like a static function – how about `this->now()`?